SharedProjects/Babylon2GLTF/GLTFExporter.Skin.cs

```diff
@@ -4,6 +4,8 @@
 using System;
 using System.Linq;
 using System.Collections.Generic;
+using Autodesk.Max;
```

Comment:
Why are we referencing Max components in the shared code?

Reply:
it was an unused namespace reference, sorry I forget it there.
It has been removed now

Reply:
Cool, we fixed the build break. :)